lucene/core/src/java/org/apache/lucene/index/FieldInfos.java

```diff
@@ -452,7 +465,8 @@ synchronized int addOrGet(FieldInfo fi) {
                 new FieldVectorProperties(
                     fi.getVectorDimension(),
                     fi.getVectorEncoding(),
-                    fi.getVectorSimilarityFunction()));
+                    fi.getVectorSimilarityFunction()),
+                new FieldMultiVectorProperties(fi.getMultiVectorSimilarityFunction()));
```

Comment:
Wondering if a separate `FieldMultiVectorProperties` is needed or if we could tack onto the `FieldVectorProperties` that is already there?
```suggestion
                    fi.getVectorSimilarityFunction(),
                    fi.getMultiVectorSimilarityFunction()));
```